test/Spring-test.js

```diff
@@ -0,0 +1,21 @@
+describe('Spring', () => {
+  const SpringInjector = require('inject!../src/Spring');
+  let stepper, utils, Spring;
+
+
+  beforeEach(() => {
+    stepper = jasmine.createSpy('stepper');
```

Comment:
Mock all internal dependencies.

Not sure about these tools, maybe we don't need to mock them. I personally prefer to mock all internal libs and utils though.